### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by misrab**
Small typo fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Plasma chain itself has very simple structure with assets being undividable and
...
+Header is submitted by Plasma network operator to the smart-contract on Ethereum chain. Blocks can only be sent one by one, with sequence numbering is enforced by contract. Any user of Ethereum network can deposit ETH to contract that will trigger and event and will allow Plasma network operator to make a funding transaction in Plasma chain. Then users can freely transact in Plasma chain, with headers pushed to parent contract in Ethereum.
 
 When user wants to settle one of his transactions to the main network, he starts a withdraw on Ethereum network by providing reference to the transaction (in a form of BlockNumber, TxNumberInBlock, OutputNumberInTX), full transaction and Merkle proof that this transaction was indeed included in that block. Parent contract checks a proof versus submittet root hash for this block and if it passed starts withdraw process. After 24 hours it can be finalized. There is a special kind of transaction in Plasma netword that can speedup a process by effectively burning the input (sending it to 0x0). If this block is not published by operator, withdraw can go as normal.
```
Comment (RyanHendricks) [inline README.md:42]: Perhaps include **submittet > submitted** on 3rd to last line and **netword > network** on 2nd to last line. Didn't want to to submit another PR for something so similar but just thought I would add these suggestions.
This is a great project!
Cheers.

**Comment by shamatar** [comment]
Good day, Ryan. 

Thank you for corrections, I should definitely include spell-checking is my routine.

Sincerely, Alexander

**Comment by RyanHendricks** [comment]
Alex, my pleasure and in all honesty the potential of the project far outweighs any need to devote energy to such trivial tasks such as spell-checking. Especially since it is open-source and there will be plenty of people willing to handle such things :). All the best, Ryan
